recipe/conda_build_config.yaml

```diff
@@ -7,6 +7,7 @@ c_compiler_version:            # [unix]
   - 12                         # [linux]
   - 16                         # [osx]
   - 11                         # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux]
+  - 12                         # [os.environ.get("CF_CUDA_ENABLED", "false") == "true" and linux]
```

Comment:
I overlooked one of the places when taking over https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5799/commits/1cf8857b12906e16641423e5572d5ae0b164e2ae where you had put in lower-case booleans, sorry.

Fixed in https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/23be5a0477d40f737eb6d15423713b277ec0788c

Reply:
No worries. Thanks Axel! 🙏